CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,8 @@
 
 - [REST API] Add backup repository, storage repository, alarms and backups jobs information in the `/rest/v0/dashboard` endpoint (PR [#7882](https://github.com/vatesfr/xen-orchestra/pull/7882), [#7904](https://github.com/vatesfr/xen-orchestra/pull/7904)), [#7914](https://github.com/vatesfr/xen-orchestra/pull/7914)
 - [SR/Disks] Show and edit the use of CBT (Change Block Tracking) [#7786](https://github.com/vatesfr/xen-orchestra/issues/7786) (PR [#7888](https://github.com/vatesfr/xen-orchestra/pull/7888))
+- [Plugin/backup-reports] Send more concise backup reports to slack and xmpp webhooks (PR [#7932](https://github.com/vatesfr/xen-orchestra/pull/7932))
```

Comment:
Use correct capitalization for [Slack](https://en.wikipedia.org/wiki/Slack_(software)) and [XMPP](https://en.wikipedia.org/wiki/XMPP).